### ISSUE TITLE
Way to turn off notifications about loaded system

### ISSUE THREAD

**Issue by apoapostolov** [edzillion/blood-n-guts#226]
Those are great during debug but for every session it gets annoying to see them. I would like to have a way to turn that notification without having to CustomCSS and display:none all notifications.

**Comment by edzillion** [comment]
Which ones are you getting? I really only see one per session, when fist loading, which alerts the user to the fact that BnG supports the running system, which I am rather in favour of keeping.

**Comment by CasualTerror** [comment]
It's one thing if the GM sees the notification when loading into the session but my players are seeing the notification too. It wasn't once during the session it was like every time we moved to a new scene and then again 30 or so minutes later. I had to turn off the module because they didn't like the notifications.

**Comment by GambetTV** [comment]
Yeah, I just came to post about this too. Literally any time my players switch scenes they get a minimum of two popup messages.

The first says: "Note: Blood n' Guts requires the first DM to be online to function!" 
The second says: "GM Present: Blood n' Guts is now Functional."

The fact that my players see both messages simultaneously means there's probably something off about how or when the system decides to fire these alerts off. But literally any time they refresh the window they get both of those messages AND a third message about compatibility with the system.

I can understand the motivation behind having all these alerts, but even if they worked perfectly I'd prefer never seeing them at all. I run with 100+ modules and this is the only one that posts popup alerts. I also only run D&D5e, so I don't see any benefit in being told the module is compatible with that every time I load the game up.

I agree with the OP, a toggle we could set to turn off the alerts completely would be a VERY welcome addition! Appreciate it!

**Comment by edzillion** [comment]
ok that's definitely excessive. I will have a look at refining the notifications now

**Comment by Jonwh25** [comment]
Yeah I'm with everyone else. I'd just rather have it be an option that I can turn that message on or off. My players also were complaining.
